guides/source/association_basics.md

```diff
@@ -563,23 +563,23 @@ Associations are extremely useful, but they are not magic. You are responsible f
 When you declare a `belongs_to` association, you need to create foreign keys as appropriate. For example, consider this model:
 
 ```ruby
-class Order < ActiveRecord::Base
-  belongs_to :customer
+class Book < ActiveRecord::Base
+  belongs_to :author
 end
 ```
 
-This declaration needs to be backed up by the proper foreign key declaration on the orders table:
+This declaration needs to be backed up by the proper foreign key declaration on the books table:
 
 ```ruby
-class CreateOrders < ActiveRecord::Migration
+class CreateBooks < ActiveRecord::Migration
   def change
-    create_table :orders do |t|
-      t.datetime :order_date
-      t.string   :order_number
-      t.integer  :customer_id
+    create_table :books do |t|
+      t.datetime :published_at
+      t.string   :book_number
```

Comment:
@maclover7  Objective was use to `book_number` that we can categories books according to category like Agriculture, sports etc.